docker-compose.yml

```
@@ -0,0 +1,284 @@
+    driver: local
+  
+  claudeops_grafana:
+    driver: local
+
+# Network configuration
+networks:
+  claudeops-network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.20.0.0/16
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)